### PR TITLE
Added a License Export function and button

### PR DESCRIPTION
**Pull request by Godmartinz**
# Description

This adds a license export button to the License index page.
Button placement:
image

exported csv:
image

Fixes #[sc-14529]

## Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* PHP version:
* MySQL version
* Webserver version
* OS version

# Checklist:

- [ ] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [ ] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Godmartinz** [comment]
the picture is for ants. sorry.

**Review by marcusmoore — APPROVED** [review]
No issues with the code but I noticed we're not exporting the "total" and "avail" columns. Not sure if that is outside of the scope of this or if it should be included.

File: app/Http/Controllers/Licenses/LicensesController.php [hunk 1]
```diff
@@ -289,4 +290,99 @@ public function getClone($licenseId = null)
...
+    public function getExportLicensesCsv()
+    {
+        $this->authorize('view', License::class);
+        \Debugbar::disable();
+
+        $response = new StreamedResponse(function () {
+            // Open output stream
+            $handle = fopen('php://output', 'w');
+
+            License::with('company','manufacturer','category', 'supplier', 'adminuser')
```
Comment (snipe) [inline app/Http/Controllers/Licenses/LicensesController.php]: I don't see any company scoping in here.

File: resources/lang/en-US/admin/licenses/table.php [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
     'assigned_to'   	=> 'Assigned To',
     'checkout'   		=> 'In/Out',
+    'deleted_at'        => 'Deleted at',
```
Comment (snipe) [inline resources/lang/en-US/admin/licenses/table.php:7]: These strings all exist in other files - we don't need to duplicate them here.

**Review by snipe — CHANGES_REQUESTED** [review]
A few small changes

File: app/Http/Controllers/Licenses/LicensesController.php (hunk 1 above)
Comment (Godmartinz) [reply app/Http/Controllers/Licenses/LicensesController.php]: I think I pushed this  before your Company scope fix 😅 --anywho its added and tested. 👍

File: resources/lang/en-US/admin/licenses/table.php (hunk 2 above)
Comment (Godmartinz) [reply resources/lang/en-US/admin/licenses/table.php:7]: the created by and updated at were duplicates, they have been removed. I do see ->deleted_at a lot but we dont have a translation string for it yet.

**Pull request merged**